src/webapp/src/components/GitProtocolPicker.js

```diff
@@ -0,0 +1,32 @@
+import RadioGroup from "@material-ui/core/RadioGroup";
+import FormControlLabel from "@material-ui/core/FormControlLabel";
+import Radio from "@material-ui/core/Radio";
+import {Title} from "./OnPremConnectionInput.style";
+import React, {useEffect, useState} from "react";
+
+export const GitProtocolPicker = ({store}) => {
+    const [gitProtocol, setGitProtocol] = useState(store.get("gitProtocol") ?? "0");
```

Comment:
Why do we use the numbers ("0","1","2")? and not strings ("http","ssh","none"/"default")?

Reply:
enum. maybe I should redefine it here though